src/context.ts

```diff
@@ -280,7 +280,7 @@ export class Context {
     this.res = new Response(null, {
       status: status,
       headers: {
-        ...this.res.headers.toJSON(),
+        ...Object.fromEntries(this.res.headers),
```

Comment:
`toJSON()` in bun about 10x faster than `Object.fromEntries(headers.entries())`

Reply:
It may be, but that usage fails pipeline for TS checking

Reply:
interesting should't be fail can you rollback `toJSON` and retry

Reply:
I have added back and pipeline is failed again. Could you look that?

Reply:
just rollback all `toJSON` i can force pipeline it should't be fail